Decide/src/main/java/DD2480/decide/CMVHandler.java

```diff
@@ -253,7 +257,20 @@ private boolean licThirteen() {
 	}
 
 	private boolean licFourteen() {
-		// TODO Auto-generated method stub
-		return false;
-	}	
+        boolean greater = false;
```

Comment:
You should check if the parameters are valid:

This most be true NUMPOINTS >= 5 and  0 < AREA2